packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerBehavior.ts

```diff
@@ -18,4 +26,7 @@ export const datepickerBehavior: Accessibility<DatepickerBehaviorProps> = props
   },
 });
 
-export type DatepickerBehaviorProps = never;
+export type DatepickerBehaviorProps = {
+  'aria-labelledby'?: string;
+  'aria-invalid'?: boolean;
```

Comment:
Let's reuse `AccessibilityAttributes`:

```suggestion
  'aria-labelledby'?: AccessibilityAttributes['aria-labelledby'];
  'aria-invalid'?: AccessibilityAttributes['aria-invalid'];
```